style.css

```diff
@@ -0,0 +1,135 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  font-size: 20px;
+}
+
+/* Mobile */
+.header {
+  height: 120px;
+  background-color: rgb(60 58 57);
+  position: relative;
+  border: 2px solid rgb(255 107 0);
+}
+
+.lo {
+  width: 100px;
+  height: 45px;
+  font-family: Georgia, "Times New Roman", Times, serif;
+  font-weight: 700;
+  font-size: 25px;
+  line-height: 30px;
+  color: white;
+  text-align: center;
+  margin-bottom: 20px;
+  position: absolute;
+  left: 40px;
+  top: 75px;
+}
+
+.img {
+  border: 2px solid #acb7c3;
+  width: 45px;
+  height: 2px;
+  position: absolute;
+  right: 90px;
+  background-color: white;
+  margin-top: 10px;
+}
+
+.img1 {
+  top: 65px;
+}
+
+.img2 {
+  top: 75px;
+}
+
+.img3 {
+  top: 85px;
+}
+
+.top {
+  position: absolute;
+  width: 54px;
+  height: 16px;
+  left: 40px;
+  top: 25px;
+  letter-spacing: 2px;
+  font-family: Georgia, "Times New Roman", Times, serif;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 16px;
+  text-align: center;
+  color: white;
+}
+
+.ico {
+  position: absolute;
+  right: 70px;
+  top: 25px;
+  display: flex;
+  gap: 5px;
+}
+
+.fa {
+  color: white;
+}
+
+.section1 {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 15px;
+  background: #1c1a19;
+  padding: 120px 64px;
+  position: relative;
+}
```

Comment:
- [ ] You did a good job using flex-box to align the content in this section ✨, however, I don't see the background image for this section:

**Your design**
![image](https://user-images.githubusercontent.com/43172164/198117952-dbd69dd5-5d6c-4c82-9995-00c827f32c20.png)

**Template design**
![image](https://user-images.githubusercontent.com/43172164/198118094-7da75480-1a45-4a42-a66b-695d8e927769.png)

- Make sure your page has the proper background image (decorations are always background images, not `img` tags). Example `background-image: url("paper.gif");` 🧱